apps/language_models/scripts/vicuna.py

```diff
@@ -127,20 +132,30 @@ def brevitas〇matmul_rhs_group_quant〡shape(lhs: List[int], rhs: List[int], rh
         raise ValueError("Input shapes not supported.")
 
 
-def brevitas〇matmul_rhs_group_quant〡dtype(lhs_rank_dtype: Tuple[int, int], rhs_rank_dtype: Tuple[int, int], rhs_scale_rank_dtype: Tuple[int, int], rhs_zero_point_rank_dtype: Tuple[int, int], rhs_bit_width: int, rhs_group_size: int) -> int:
```

Comment:
pls revert the format

Reply:
reverted in https://github.com/nod-ai/SHARK/pull/1702/commits/7c35d276c4a14e3a52de8c09b8a681cc86cc9777